From e9ec7f11712aa0f8a10285722cfec2f51e05d230 Mon Sep 17 00:00:00 2001
From: Tzafrir Cohen <tzafrir@debian.org>
Date: Fri, 19 Apr 2013 12:58:47 +0300
Subject: [PATCH] remove unused libmilenage

Milenage is a 3GPP authentication algorithm.
libmilenage was linked in but was not actually used. Just remove it for
now.
---
 build.mak.in                   |    8 +-------
 pjsip/src/pjsip/sip_auth_aka.c |    2 --
 2 files changed, 1 insertion(+), 9 deletions(-)

--- a/build.mak.in
+++ b/build.mak.in
@@ -26,13 +26,7 @@ endif
 # Determine which party libraries to use
 export APP_THIRD_PARTY_EXT :=
 export APP_THIRD_PARTY_LIBS :=
-export APP_THIRD_PARTY_LIB_FILES := $(PJ_DIR)/third_party/lib/libmilenage-$(LIB_SUFFIX)
-ifeq ($(PJ_SHARED_LIBRARIES),)
-APP_THIRD_PARTY_LIBS += -lmilenage-$(TARGET_NAME)
-else
-APP_THIRD_PARTY_LIBS += -lmilenage
-APP_THIRD_PARTY_LIB_FILES += $(PJ_DIR)/third_party/lib/libmilenage.$(SHLIB_SUFFIX).$(PJ_VERSION_MAJOR) $(PJ_DIR)/third_party/lib/libmilenage.$(SHLIB_SUFFIX)
-endif
+export APP_THIRD_PARTY_LIB_FILES :=
 
 ifeq (@ac_external_srtp@,1)
 # External SRTP library
--- a/pjsip/src/pjsip/sip_auth_aka.c
+++ b/pjsip/src/pjsip/sip_auth_aka.c
@@ -29,8 +29,6 @@
 
 #if PJSIP_HAS_DIGEST_AKA_AUTH
 
-#include "../../third_party/milenage/milenage.h"
-
 /*
  * Create MD5-AKA1 digest response.
  */
--- a/third_party/build/Makefile
+++ b/third_party/build/Makefile
@@ -1,4 +1,4 @@
-DIRS = milenage
+DIRS =
 
 include ../../build.mak
 include $(PJDIR)/build/common.mak
